src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -1,4 +1,4 @@
-<div>
+<div xmlns="http://www.w3.org/1999/html">
```

Comment:
Any reason why the XML namespace is added here? Do revert this if it's not necessary.

Reply:
This is not useful, i have reverted it